source/common/http/http3/conn_pool.cc

```diff
@@ -72,10 +73,16 @@ Http3ConnPoolImpl::Http3ConnPoolImpl(
       host->cluster().perConnectionBufferLimitBytes());
 }
 
+void Http3ConnPoolImpl::onConnected(Envoy::ConnectionPool::ActiveClient&) {
+  if (connect_callback_ != absl::nullopt) {
```

Comment:
In the case where the upstream connection is using 0-RTT will this method fire when the handshake completes or when 0-RTT data is sendable?

Reply:
If we're simply trying to be informed when the Network::Connection is Connected, why not pass down a ConnectionCallbacks& to be called on the Connected event? I'm fine if we want to have a similar API for 0-rrt connected vs fully connected but if that's the case maybe we should comment that somewhere?